### ISSUE TITLE
[Bug] compilation error on debian

### ISSUE THREAD

**Issue by werdahias** [StarLabsLtd/coreboot-configurator#17]
I am working on an official package for coreboot-configurator. I installed all the relevant dependencies. Yet the build fails: 

```
[8/17] ccache c++ -Isrc/application/coreboot-configurator.p -Isrc/application -I../src/application -I/usr/include/x86_64-linux-gnu/qt5 -I/usr/include/x86_64-linux-gnu/qt5/QtCore -I/usr/include/x86_64-linux-gnu/qt5/QtWidgets -I/usr/include/x86_64-linux-gnu/qt5/QtSvg -I/usr/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -std=c++14 -O0 -g -O2 -ffile-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -DQT_CORE_LIB -DQT_WIDGETS_LIB -DQT_SVG_LIB -fPIC -MD -MQ src/application/coreboot-configurator.p/MainWindow.cpp.o -MF src/application/coreboot-configurator.p/MainWindow.cpp.o.d -o src/application/coreboot-configurator.p/MainWindow.cpp.o -c ../src/application/MainWindow.cpp
FAILED: src/application/coreboot-configurator.p/MainWindow.cpp.o 
ccache c++ -Isrc/application/coreboot-configurator.p -Isrc/application -I../src/application -I/usr/include/x86_64-linux-gnu/qt5 -I/usr/include/x86_64-linux-gnu/qt5/QtCore -I/usr/include/x86_64-linux-gnu/qt5/QtWidgets -I/usr/include/x86_64-linux-gnu/qt5/QtSvg -I/usr/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -std=c++14 -O0 -g -O2 -ffile-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -DQT_CORE_LIB -DQT_WIDGETS_LIB -DQT_SVG_LIB -fPIC -MD -MQ src/application/coreboot-configurator.p/MainWindow.cpp.o -MF src/application/coreboot-configurator.p/MainWindow.cpp.o.d -o src/application/coreboot-configurator.p/MainWindow.cpp.o -c ../src/application/MainWindow.cpp
../src/application/MainWindow.cpp:8:10: fatal error: QtGui: No such file or directory
    8 | #include <QtGui>
      |          ^~~~~~~
compilation terminated.

```

**Comment by Sean-StarLabs** [comment]
Whats the version of `qt`?

**Comment by werdahias** [comment]
Sorry, forgot to mention:
OS: debian sid
libqt5gui5, qtbase5-dev version: 5.15.6

**Comment by werdahias** [comment]
This is indeed a bug: 

`Gui` isn't in `src/application/meson.build`.  This patch solves that:

``diff
--- coreboot-configurator-9.orig/src/application/meson.build
+++ coreboot-configurator-9/src/application/meson.build
@@ -2,7 +2,7 @@
 
 # Documentation: https://mesonbuild.com/Qt5-module.html
 qt5 = import('qt5')
-qt5_dep = dependency('qt5', modules : ['Core', 'Widgets', 'Svg'])
+qt5_dep = dependency('qt5', modules : ['Core', 'Widgets', 'Svg', 'Gui'])
 yamlcpp_dep = dependency('yaml-cpp', version: '>= 0.5.1', required: true)
 



```

**Comment by Sean-StarLabs** [comment]
Thank you! Merged 944b575dc873c78627c352f9c1a1493981431a58
